--- libextractor-java-0.5.6/org/gnunet/libextractor/Makefile.am.orig	2005-10-10 18:51:03.000000000 +0200
+++ libextractor-java-0.5.6/org/gnunet/libextractor/Makefile.am	2005-10-28 21:20:33.568473472 +0200
@@ -8,5 +8,5 @@
   Xtract.java
 
 install-exec-local:
-	mkdir -p $(prefix)/share/java
-	cd ../../..; @JAR@ -cvf $(prefix)/share/java/libextractor.jar org/gnunet/libextractor/*.class
+	mkdir -p $(DESTDIR)$(prefix)/share/java
+	cd ../../..; @JAR@ -cvf $(DESTDIR)$(prefix)/share/java/libextractor.jar org/gnunet/libextractor/*.class
